Doc/library/datetime.rst

```diff
@@ -1813,8 +1813,6 @@ Other constructor:
 
    Examples::
 
```

Comment:
Thanks! However, if we remove the `.. doctest::` line, then the doctest won't be run as part of our CI. I think the better fix is to remove the second colon after "Examples" -- something like this?

```suggestion
   Examples:

   .. doctest::

```

Reply:
(If you like my suggestion above, you can click the "commit suggestion" button immediately beneath the comment to apply the suggestion to your PR branch :)

Reply:
Oh, sorry, my bad

Reply:
No need to apologise!